### DISCUSSION TITLE
How to understand classes?

### DISCUSSION THREAD

**Discussion by Whhhatttt** [TomSchimansky/CustomTkinter#1473] [Q&A]
I know that the use of classes simplifies the style of the code, and also allows you to simplify the whole code. But I don't understand how to do it. How many written programs I looked at - I still don't understand how to write classes, how to use them. For me, this is a dark forest.

If you have any articles about this, or can tell me how to use classes - I will be very grateful.

**Comment by avalon60** [comment]
This should get you started:

https://youtu.be/apACNr7DC_s
and possibly...
https://www.youtube.com/watch?v=f0TrMH9s-VE&t=201s

There are also plenty of articles around if you Google it. 

They can get a little involved, but start with the basics and then move onto stuff like super-classes, decorators etc later on.

**Comment by Wolf-SO** [comment]
@DimasRulit To give you a quick idea that will change your perspective. 

Think of classes as lightweight cooperative programs.
